apps/wallet-ui/src/app/layout.tsx

```diff
@@ -15,8 +18,12 @@ export default function RootLayout({
   children: React.ReactNode;
 }>) {
   return (
-    <html lang="en">
-      <body className={inter.className}>{children}</body>
+    <html lang="en" suppressHydrationWarning>
```

Comment:
why is this needed?

Reply:
Because hydration errors annoy me but it should be removed because they should all be gone now